docs/guidance/map/framework_agreements.md

```diff
@@ -63,7 +63,7 @@ In some cases, complex contracting processes cannot be represented under a singl
 * There are multiple competitive stages: for example, when a framework agreement involves second-stage competitions.
 * The procurement systems used at different stages of the process are managed by different bodies, and cannot be integrated.
 
-OCDS models the first and second stages of framework agreement procedures as separate contracting processes, linked together using the `relatedProcesses` array. The `tender.techniques.hasFrameworkAgreement` field, from the [Techniques](https://extensions.open-contracting.org/en/extensions/techniques/master/) extension, is used to identify contracting processes that represent the first stage of a framework agreement procedure. The presence of a related process with a `.relationship` set to 'framework' is used to identify contracting processes that represent the second stage of a framework agreement procedure.
+OCDS models the first and second stages of framework agreement procedures as separate contracting processes, linked together using the `relatedProcesses` array. The `tender.techniques.hasFrameworkAgreement` field, from the [Techniques](https://extensions.open-contracting.org/en/extensions/techniques/master/) extension, is used to identify contracting processes that represent the first stage of a framework agreement procedure. The presence of a related first-stage process with a `.relationship` set to 'framework' is used to identify contracting processes that represent the second stage of a framework agreement procedure. Second-stage contracting processes ought to be always related to the first-stage contracting process, never to other second-stage contracting processes.
```

Comment:
I think it's important to preserve the word "related process" and not "related first-stage process" because "related process" is more obviously connected to `relatedProcesses`.

```suggestion
OCDS models the first and second stages of framework agreement procedures as separate contracting processes, linked together using the `relatedProcesses` array. The `tender.techniques.hasFrameworkAgreement` field, from the [Techniques](https://extensions.open-contracting.org/en/extensions/techniques/master/) extension, indicates whether the contracting process represents the first stage of a framework agreement procedure. The presence of a related process with a `.relationship` field set to 'framework' indicates whether the contracting process represents the second stage of a framework agreement procedure.

.. note::

   The 'framework' code is only used to relate a second-stage contracting process to its first-stage contracting process, never to other second-stage contracting processes.
```

Reply:
Generally better, thanks. The one point I would slightly defend: I think speaking about "second-stage contracting processes linked to their first-stage contracting process", i.e. using the plural, is useful/educative, because it drills in the fact that there is a one-to-many relationship there. (Which is possibly not the first thing you think of when you hear "first stage" and "second stage"). On the other hand, I understand we try to speak in the singular.

I've proposed to put the plural back in https://github.com/open-contracting/standard/pull/1461/commits/f5d3209655b8549c223108b84a775a804c7a11c4.

Reply:
Yes, sounds good.